package.json

```diff
@@ -118,6 +118,8 @@
     "@babel/preset-react": "~7.9.1",
     "@babel/register": "~7.9.0",
     "@babel/runtime-corejs3": "~7.9.0",
+    "@types/fetch-mock": "^7.3.5",
```

Comment:
why do we need these two packages?

Reply:
These were from old test cases, I think I can remove them though